maintainers/maintainer-list.nix

```diff
@@ -14876,4 +14876,10 @@
     github = "npatsakula";
     githubId = 23001619;
   };
+  xyenon = {
+    name = "XYenon";
+    email = "i@xyenon.bid";
+    github = "xyenon";
+    githubId = 20698483;
+  };
```

Comment:
Sorry to be so nitpicky, though there are some other entries ignoring this request, it will still be nice to keep the list alphabetically sorted:

https://github.com/NixOS/nixpkgs/blob/56a682817d68050298b9fb0c56628b5a85441ad4/maintainers/maintainer-list.nix#L49

Reply:
Sorry for that. I will fix it soon.

Reply:
Fixed.